lib/board.rb

```diff
@@ -0,0 +1,66 @@
+class Board
+
+	attr_accessor :cells
+
+	def initialize
+		reset!
+	end
+
+	def reset!
+		@cells = Array.new(9, " ")
+	end
+
+	def display
+		puts " #{cells[0]} | #{cells[1]} | #{@cells[2]} "
+		puts "-----------"
+		puts " #{cells[3]} | #{cells[4]} | #{@cells[5]} "
+		puts "-----------"
+		puts " #{cells[6]} | #{cells[7]} | #{@cells[8]} "
+	end
+
+	def position(input)
+		@cells[(input.to_i - 1)]
+	end
+
+	def full?
+		if @cells.include?(" ")
+			false
+		else
+			true
+		end
+	end
```

Comment:
Avoid using the ternary operator or if/else to return true and false. Instead, rely on the implicit boolean return value of the expression.

cells.include?(" ")

This applies to `#taken?` also.  In addition, it's better practice to access instance variables and attributes through their getter and setter methods rather than directly, in this case, `cells` instead of `@cells`.